docs/quickstart.rst

```diff
@@ -61,69 +147,67 @@ instance had already existing in the destination directory, this command
-
-In the example above, the keeper creates a primary database. It chooses
-to set up node A as primary because the monitor reports there are no
-other nodes in the system yet. This is one example of how the keeper is
-state-based: it makes observations and then adjusts its state, in this
-case from "init" to "single."
-
-At this point the monitor and primary nodes are created and
-running. Next we need to run the keeper. It’s an independent process so
-that it can continue operating even if the Postgres primary goes down:
+     --pgdata ha \
+     --pgctl `which pg_ctl` \
+     --nodename `hostname -I` \
+     --monitor postgres://autoctl_node@10.0.1.4/pg_auto_failover
+
+Notice the user and database name in the monitor connection string -- these
+are what monitor init created. We also give it the path to pg_ctl so that the
+keeper will use the correct version of pg_ctl in future even if other versions
+of postgres are installed on the system.
+
+In the example above, the keeper creates a primary database. It chooses to set
+up node A as primary because the monitor reports there are no other nodes in
+the system yet. This is one example of how the keeper is state-based: it makes
+observations and then adjusts its state, in this case from "init" to "single."
+
+At this point the monitor and primary nodes are created and running. Next we
+need to run the keeper. It’s an independent process so that it can continue
+operating even if the Postgres primary goes down:
 
 .. code-block:: bash
 
-  pg_autoctl run --pgdata /mnt/node_a/data
+   # run this on the node A virtual machine as well
+
+   pg_autoctl run --pgdata ha
 
-This will remain running in the terminal, outputting logs. We can open
-another terminal and start a secondary database similarly to how we
-created the primary:
+This will remain running in the terminal, outputting logs. Next connect to
+node B and do the same process.
 
 .. code-block:: bash
 
-   pg_autoctl create postgres  \
-     --pgdata ./node_b      \
-     --pgport 6011          \
-     --pgctl /usr/pgsql-11/bin/pg_ctl \
-     --monitor postgres://autoctl_node@127.0.0.1:6000/pg_auto_failover
+   # on the node B virtual machine
 
-   pg_autoctl run --pgdata ./node_b
+   sudo su - postgres
+   export PATH="/usr/lib/postgresql/11/bin/:$PATH"
 
-All that differs here is we’re running it on another port and pointing
-at another data directory. It discovers from the monitor that a primary
-exists, and then switches its own state to be a hot standby and begins
-streaming WAL contents from the primary.
+   pg_autoctl create postgres     \
+     --pgdata ha \
+     --pgctl `which pg_ctl` \
+     --nodename `hostname -I` \
+     --monitor postgres://autoctl_node@10.0.1.4/pg_auto_failover
+
+   pg_autoctl run --pgdata ha
+
+It discovers from the monitor that a primary exists, and then switches its own
+state to be a hot standby and begins streaming WAL contents from the primary.
```

Comment:
I think now is a good time to add a section about the Postgres HBA file and setup, and show the HBA entries created by pg_autoctl in every node. We would have two goals with such a section:

  1. explain what pg_autoctl is doing automatically behind the scene to make everything work
  2. explain that those are only defaults and you might want to use your own setting either on-top of it, or instead of it...

We don't have the `--skip-hba` option yet but we want to get there someday. People might want to use kerberos or LDAP based authentication, or even full SSL certificates with client and server validation. On the cloud, I would argue it's a good idea. We don't automate that, and should encourage users to understand their options and consider doing that for their production installation.

Reply:
Also, this is again a point where I believe that showing the output of the pg_autoctl command would be enlightening for our readers.